openstack_controller/datadog_checks/openstack_controller/api/api_rest.py

```diff
@@ -128,6 +128,14 @@ def get_compute_quota_set(self, project_id):
             return component.get_quota_set(project_id)
         return None
 
+    def get_compute_services(self, project_id):
```

Comment:
Methods added to this class need to be abstract in base class

Reply:
Updated!